pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -67,7 +67,7 @@ type event struct {
 	gvk    schema.GroupVersionKind
 }
 
-// GraphBuilder: based on the events supplied by the informers, GraphBuilder updates
+// GraphBuilder based on the events supplied by the informers, GraphBuilder updates
```

Comment:
How about "GraphBuild processes events supplied by the informers, updates uidToNode, ..."?

Reply:
There is a comment that starts with GraphBuilder, but golint outputs error:
pkg/controller/garbagecollector/graph_builder.go:70:1: comment on exported type GraphBuilder should be of the form "GraphBuilder ..." (with optional leading article)
This error is caused by ":" in the comment, so remove it.

Reply:
Simply removing ":" broke the sentence, that's why I suggested the change.

Reply:
I got the point and updated commit.
Could you check it?